src/utils/hashes/index.ts

```diff
@@ -79,76 +116,79 @@ export const computeTransactionHash = (txJSON: any): string => {
  * @param txBlobHex - The binary transaction blob as a hexadecimal string.
  * @returns The hash to sign.
  */
-export const computeBinaryTransactionSigningHash = (
-  txBlobHex: string
-): string => {
+export function computeBinaryTransactionSigningHash(txBlobHex: string): string {
   const prefix = HashPrefix.TRANSACTION_SIGN.toString(16).toUpperCase();
   return sha512Half(prefix + txBlobHex);
-};
+}
 
 /**
- * Compute Account Root Index.
+ * Compute Account Ledger Object ID.
```

Comment:
```suggestion
 * Compute AccountRoot Ledger Object ID.
```
and so on with the rest of the docs